### DISCUSSION TITLE
ScriptReference/PrefabUtility.IsDefaultOverride

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#643] [Notes]
# ScriptReference/PrefabUtility.IsDefaultOverride

https://docs.unity3d.com/ScriptReference/PrefabUtility.IsDefaultOverride.html

**Comment by STARasGAMES** [comment]
Order of serialized properties and their names:
m_LocalRotation
m_LocalPosition
m_LocalEulerAnglesHint

private static string[] TransformDefaultOverrideProperties = new[] { "m_LocalRotation", "m_LocalPosition", "m_LocalEulerAnglesHint" };
